itemexists.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"reflect"
+)
+
+//check if any item exists or not
+func main() {
+	var strSlice = []string{"ozombo", "twilio", "unicodedeveloper", "amycruz", "codebeast"}
```

Comment:
The contents should contain actual words not the Monica or names of people. To make this example practical use fruits

```go
func main () {
    fruits := [string{"apple", "banana", "mango"}
}

refactor to fit this

Reply:
@ozombo